File: examples/currency/codes/go/getSupportedCodes.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/currency"
	"os"
)

// Codes returns the supported currency codes for the API
func GetSupportedCodes() {
	currencyService := currency.NewCurrencyService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := currencyService.Codes(&currency.CodesRequest{})
	fmt.Println(rsp, err)
}

File: examples/currency/codes/node/getSupportedCodes.js
const { CurrencyService } = require("m3o/currency");

// Codes returns the supported currency codes for the API
async function getSupportedCodes() {
  let currencyService = new CurrencyService(process.env.MICRO_API_TOKEN);
  let rsp = await currencyService.codes({});
  console.log(rsp);
}

getSupportedCodes();
